src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -712,6 +715,7 @@ export class List<T> implements ISpliceable<T>, IDisposable {
 	setFocus(indexes: number[]): void {
 		indexes = indexes.sort(numericSort);
 		this.focus.set(indexes);
+		this.view.domNode.setAttribute('aria-posinset', indexes.length ? (indexes[0] + 1).toString() : undefined);
```

Comment:
The [w3 docs](https://www.w3.org/TR/wai-aria/states_and_properties#aria-posinset) state:

> Authors MUST set the value for aria-posinset to an integer greater than or equal to 1, and less than or equal to the size of the set.

Setting it to `undefined` doesn't sound like it's the right thing to do here.